### DISCUSSION TITLE
Is it time to refresh the WinUI Logo?

### DISCUSSION THREAD

**Discussion by mdtauk** [microsoft/microsoft-ui-xaml#8645] [Ideas]
Looking at the latest set of Emoji's, Fluent Icons, and WinUI Gallery icons.
I am thinking it may be time to give the WinUI logo a refresh. 

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/7389110/01331c1d-10eb-4d2a-b13f-17d3673626b5)

I feel like the older logo I designed, is letting the side down as we move towards more Fluent 2 styles, and a brighter and more colourful approach may be needed.

**As the person who designed the current logo, I thought I would start the conversation, and show some of my thinking.**

Here is a design I feel keeps much of the old, but moves it forward.  I present it to you in various colours.

One set from the Blend, VS Code, and Visual Studio palette.
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/7389110/3d07cc11-738f-498b-ac65-cd6a299e3b0d)

Some with the original palettes of Blues.
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/7389110/a0fff698-c2de-4d17-a85d-b3546e98facc)

And finally my favourite, is using the colour scheme for the icons in the WinUI Gallery.
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/7389110/44af99ad-14ce-45d9-84cb-c2f7b18601bb)

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/7389110/7e28c89d-35aa-44e2-be4b-c00345516428)

**Comment by saint4eva** [comment]
+1

**Comment by mdtauk** [comment]
I think @chigy was announced as the new leader of the WinUI project, so maybe this is a decision for her?  I think people are returning from vacations so maybe this can be discussed either within the team, or during the next WinUI Community Call - if those will still be taking place in the future, now WinUI is becoming more integrated into the WinAppSDK project.

**Comment by chigy** [comment]
Hello @mdtauk ,
We are working together as a team to better the situation in GitHub as was announced in the last community call. Getting the issues number down and having our processes better explained to the community members, etc. We are currently heads-down on these and other time sensitive tasks such as planning for 1.5 release and are not thinking about rebranding ourselves.

**Reply by weitzhandler** [reply]
That was one and a half years ago and not a lot changed since.
The more important thing for WinUI is fixing its long overdue critical bugs and outstanding issues. WinUI suffers from tremendous neglect and disrespect from Microsoft.

**Comment by jamespack** [comment]
The one with the orange gives Firefox vibes

**Comment by llothar** [comment]
I like the cube and i'm  tired of all this simplification of logos.

**Comment by mdtauk** [comment]
It looks like version 1.6 or 1.7 is bringing an updated logo.

I'm glad this has been addressed, and it looks more fitting among the fluent logo style that has spread across Windows 11 and other Microsoft branding.

![image](https://github.com/user-attachments/assets/63ea85df-3c87-4d26-8eee-cb2babb0347d)

**Reply by niels9001** [reply]
@mdtauk Yup 😁! We've been working with the Windows Design team to create a consistent set of icons. We'll be rolling these out over the coming weeks across the different touchpoints!

![Frame 2117129015](https://github.com/user-attachments/assets/ebc78472-8a13-4ea2-b99b-fd9a888aaf82)

**Comment by haidar47x** [comment]
No need. This thing looks abandoned already.
